src/ripple/beast/utility/rngfill.h

```diff
@@ -21,6 +21,7 @@
 #define BEAST_RANDOM_RNGFILL_H_INCLUDED
 
 #include <array>
+#include <assert.h>
```

Comment:
Please use `<cassert>` and not `<assert.h>`. Thanks.

Reply:
`<assert.h>` is also used in `MathUtilities.h`. 

I'll go ahead and change that one to `<cassert>` as well.